todo-frontend-angular/src/app/todo/button/my-button.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './my-button.component.html',
+  styleUrls: ['./my-button.component.scss']
+})
+export class ButtonComponent implements OnInit {
+	
+  @Input() buttonConfig: any;
```

Comment:
Lesson 127 in typescript :D Always use types!

Create a type/interface for the button config and use that!